docs/pages/server-access/guides/azure-discovery.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Configure Teleport to Automatically Enroll Azure Virtual Machines (Preview)
+description: How to configure Teleport to automatically enroll Azure Virtual Machines.
+---
+
+The Teleport Discovery Service can connect to Azure and automatically
+discover and enroll virtual machines matching configured labels. It will then
+execute an install script on these discovered instances that will install
+Teleport, start it and join the cluster.
+
+In this scenario, Teleport Discovery Service uses an Azure invite token with a
+long time-to-live (TTL), so that new instances can be discovered and added to
+the Teleport cluster for the lifetime of the token.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- Azure subscription with virtual machines and permissions to create and attach
+managed identities.
+- Azure virtual machines running Ubuntu/Debian/RHEL if making use of the
```

Comment:
Is this for servers you'd like to join to the Teleport cluster as well as the Discovery Service?